### PR TITLE
Simple Crashlytics Service

### PR DESCRIPTION
**Pull request by Antonwy**
Instead of:
```swift
#if !DEBUG
            Crashlytics.crashlytics().record(error: error)
#endif
```

You can now write just:
```swift
CrashlyticsService.log(error)
```

### CODE REVIEW

**Review by philippzagar — APPROVED** [review]
LGTM! Thanks!

File: Campus-iOS/Crashlytics/CrashlyticsService.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+//  CrashlyticsService.swift
+//  Campus-iOS
+//
+//  Created by Anton Wyrowski on 28.01.23.
+//
+
+import Foundation
+import FirebaseCrashlytics
+
+class CrashlyticsService {
```
Comment (philippzagar) [inline Campus-iOS/Crashlytics/CrashlyticsService.swift:11]: One minor thing: One could also put the call to the config method from Crashlytics into this service

Comment (Antonwy) [reply Campus-iOS/Crashlytics/CrashlyticsService.swift:11]: I thought of this too, but the config call is `FirebaseApp.configure()`. 
So the call is not only for crashlytics, but also for all other services of firebase and therefore perhaps wrong in the crashlytics service.

Comment (philippzagar) [reply Campus-iOS/Crashlytics/CrashlyticsService.swift:11]: Seems reasonable 👍

**Pull request merged**